.env.local.example

```diff
@@ -7,13 +7,25 @@ RPC_URL_59144=https://linea-rpc.publicnode.com
 
 # Identify which indexer plugin to activate (see `src/plugins` for available plugins)
 
-ACTIVE_PLUGIN=base.eth
+# ACTIVE_PLUGINS=eth,base.eth,linea.eth
+# ACTIVE_PLUGINS=base.eth,linea.eth
+ACTIVE_PLUGINS=eth
 
 # Database configuration
 
-# This is where the indexer will create the tables defined in ponder.schema.ts
-# No two indexer instances can use the same database schema at the same time. This prevents data corruption.
-# @link https://ponder.sh/docs/api-reference/database#database-schema-rules
-DATABASE_SCHEMA=subname_index_base.eth
-# The indexer will use Postgres with that as the connection string. If not defined, the indexer will use PSlite.
+# This is a namespace for the tables that the indexer will create to store indexed data.
+# It should be a string that is unique to the running indexer instance.
+#
+# Keeping the database schema unique to the indexer instance is important to
+# 1) speed up indexing after a restart
+# 2) prevent data corruption
+#
+# No two indexer instances can use the same database schema at the same time.
+#
+# Read more about database schema rules here:
+# https://ponder.sh/docs/api-reference/database#database-schema-rules
+DATABASE_SCHEMA=ens
+
+# This is the connection string for the database that the indexer will use to store data.
+# It should be in the format of `postgresql://<username>:<password>@<host>:<port>/<database>`
```

Comment:
@lightwalker-eth I've added more context around what those env vars are meant for